jwst/tso_photometry/tests/test_tso_photometry_step.py

```diff
@@ -72,6 +75,7 @@ def test_tsophotometry_step_full_frame(log_watcher):
 
 def test_tsophotometry_step_save_catalog(tmp_path, log_watcher):
     datamodel = mock_nircam_image()
+    include_int_times(datamodel)
```

Comment:
Could consider moving this call inside of `mock_nircam_image`, with an optional flag to skip for the no int_times test - might be more readable? Otherwise, looks fine to me. 👍

Reply:
Thanks, I will make this change.  I had a hallucination and thought this mock was coming from somewhere outside of `tso_photometry/` entirely.  Looking again, nope, it's right there.  So I'll fix that.

Reply:
how does https://github.com/spacetelescope/jwst/pull/10042/commits/72e608038090290ba93499fde8d88fb68d7523cb look?  I also added coverage for the other failure cases (`int_times` is None, `int_times` is an empty list)